configuration_hpt.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python
```

Comment:
duplicate?

Reply:
Thanks for catching this! It was a bug when I clean up!